composer.json

```diff
@@ -25,13 +25,13 @@
     "require": {
         "php": "^7.0",
         "ext-sockets": "*",
-        "opentracing/opentracing": "1.0.0-beta5",
+        "opentracing/opentracing": "1.0.0-beta6",
```

Comment:
This will require change php version:
```diff
- "php": "^7.0",
+ "php": "^7.1",
```

Also need to update Travis' build matrix and README.md